src/tools/mmaps_generator/MapBuilder.h

```diff
@@ -136,6 +137,8 @@ namespace MMAP
 
             float m_maxWalkableAngle;
             bool m_bigBaseUnit;
+            uint32 m_totalTiles;
+            uint32 m_totalTilesDone;
```

Comment:
Always prefer template std::atomic<T> over std::atomic_T typedef. Not sure about current state in most recent compilers but these used to not be the same